src/Zemin.java

```diff
@@ -0,0 +1,28 @@
+public class Zemin {
+    String aciklama;
```

Comment:
Değişkenlerin erişim  belirteci yazılmamış

Reply:
Task de bununla ilgili bir şey söylenmediği için belirtmedim. 
public olarak düzelttim.